frontend/src/shared/components/tasque-button/button.component.ts

```diff
@@ -1,7 +1,7 @@
 import { Component, Input, Output, EventEmitter } from '@angular/core';
 
 @Component({
-  selector: 'app-button',
```

Comment:
I'm not sure this is good idea to change selector of already done components coz they can be used in another branches, where selector is still 'app-button'

Reply:
If that were the case, I wouldn't be able to build our app without errors. I did it as part of changing the naming to one standard